lib/truffle/rbconfig.rb

```diff
@@ -214,9 +219,11 @@ module RbConfig
     'RMDIR'             => rmdir,
     'RMDIRS'            => "#{rmdir} -p",
     'RPATHFLAG'         => ' -Wl,-rpath,%1$-s',
+    'RUBY_API_VERSION'  => "#{major}.#{minor}",
```

Comment:
Mmh, I wonder if this should be `ruby_abi_version.dup` instead.
https://github.com/search?q=RUBY_API_VERSION&type=code doesn't make it super clear, but it does seems related to `ruby_version` which is the ABI version.
So I think using `ruby_abi_version.dup` is better here, unless you know something that would break if it's not `major.minor`?

Reply:
I guess this one is meant to mean "the version of the C API", in which case the ABI version is the closest thing we have to that (at least it changes whenever header files are changed).

Reply:
It does seem like it should match this `Gem.ruby_api_version` which uses `ruby_version`:
https://github.com/ruby/ruby/blob/5e3f1b6a6d64598982219644edfa9c3da09bafed/lib/rubygems.rb#L829-L835